protos/config.proto

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+syntax = "proto3";
+
+message Config {
```

Comment:
As this will have to be compatible across versions, I would try to minimize the number of fields that we define. The vast majority of these are used in the code, but don't have a corresponding ui and that code effectively does nothing. So I would prefer to get rid of that code rather than adapting new protos to it.

Reply:
Makes sense, thank you so much, will do!